mcs/tools/al/Makefile

```diff
@@ -2,7 +2,7 @@ thisdir = tools/al
 SUBDIRS = 
 include ../../build/rules.make
 
-LOCAL_MCS_FLAGS = -r:Mono.Security.dll
+LOCAL_MCS_FLAGS = -r:Mono.Security.dll -r:System.Security.dll -r:Mono.CompilerServices.SymbolWriter
```

Comment:
You probably don't need Mono.CompilerServices.SymbolWriter, just check for rights defines